security-analysis/security-analysis-api/src/main/java/com/powsybl/security/execution/SecurityAnalysisExecutionInput.java

```diff
@@ -39,6 +40,7 @@ public class SecurityAnalysisExecutionInput {
     private final List<String> resultExtensions = new ArrayList<>();
     private final Set<LimitViolationType> violationTypes = EnumSet.noneOf(LimitViolationType.class);
     private boolean withLogs = false;
+    private final List<Scenario> scenarios = new ArrayList<>();
```

Comment:
Question:

That class is more or less related to security analysis tool, and distributed execution.
Do we update the tool to be able to input actions and strategies, or do we postpone it to a further evolution?
Some code in there is quite messy ...